suite-native/atoms/src/Sheet/BottomSheet.tsx

```diff
@@ -19,13 +19,14 @@ export type BottomSheetProps = {
     title?: ReactNode;
     subtitle?: ReactNode;
     isScrollable?: boolean;
+    footer?: ReactNode;
 } & BoxProps;
 
 type WrapperStyleProps = {
     insetBottom: number;
 };
 
-const DEFAULT_INSET_BOTTOM = Platform.OS === 'android' ? 48 : 0;
+export const DEFAULT_INSET_BOTTOM = Platform.OS === 'android' ? 48 : 0;
```

Comment:
export is not needed
```suggestion
const DEFAULT_INSET_BOTTOM = Platform.OS === 'android' ? 48 : 0;
```

Reply:
https://github.com/trezor/trezor-suite/pull/17981/commits/614ee4b4f118a0e31c642a2014823012d4b420ae